app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,9 @@
+<% provide(:title, 'About')%>
+<!DOCTYPE html>
```

Comment:
There is no need to put DOCTYPE, HTML, and HEAD inside of a child file.
Application.html.erb already has these attributes.
Please review our code in TWITTER clone or Parent-Child relationship lessons regarding Ruby on Rails .html.erb files.